datasimulator/utils.py

```diff
@@ -37,14 +37,17 @@ def get_field(d, fields):
     return result
 
 
-def generate_list_numbers_from_file(data_file, submission_order):
+def generate_list_numbers_from_file(data_file, submission_order, n_samples):
```

Comment:
I want to rename `data_file` to something like `node_num_instances_file` or rename `data` below to `nums_instances` :( ...feel free to ignore